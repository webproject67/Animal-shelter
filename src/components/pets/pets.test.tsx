import * as React from "react";
import * as renderer from "react-test-renderer";
import Pets from "./pets";

const base = [
  {name: 'Katrine',
  src: `1`}
];

it(`Render Pets`, () => {
  const root = renderer.create(<Pets 
    base={base}
    onButtonClick={() => {}}
  />).toJSON();

  expect(root).toMatchSnapshot();
});