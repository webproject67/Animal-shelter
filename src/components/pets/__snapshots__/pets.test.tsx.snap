// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Pets 1`] = `
Array [
  <header
    className="page-header page-header--background"
  >
    <div
      className="page-header__wrapper page-header__wrapper--padding"
    >
      <nav
        className="main-nav"
      >
        <p
          className="main-nav__logo main-nav__logo--color"
          onClick={[Function]}
        >
          Cozy House 
          <span
            className="main-nav__logo--font-family main-nav__logo--color1"
          >
            Shelter for pets in Boston
          </span>
        </p>
        <ul
          className="main-nav__list"
        >
          <li
            className="main-nav__item"
          >
            <a
              className="main-nav__link main-nav__link--color"
              onClick={[Function]}
            >
              About the shelter
            </a>
          </li>
          <li
            className="main-nav__item"
          >
            <a
              className="main-nav__link main-nav__link--color main-nav__link--active"
            >
              Our pets
            </a>
          </li>
          <li
            className="main-nav__item"
          >
            <a
              className="main-nav__link main-nav__link--color"
              onClick={[Function]}
            >
              To help the shelter
            </a>
          </li>
          <li
            className="main-nav__item"
          >
            <a
              className="main-nav__link main-nav__link--color"
              onClick={[Function]}
            >
              Contacts
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </header>,
  <main
    className="page-main"
  >
    <section
      className="friends friends--padding"
    >
      <h1
        className="friends__title"
      >
        Our friends who 
        <br />
         are looking for a house
      </h1>
      <ul
        className="friends__list friends__list--width"
      >
        <li
          className="friends__item friends__item--margin"
          onClick={[Function]}
        >
          <img
            alt="Katrine"
            className="friends__img"
            src="img/desktop/1@1x.jpg"
            srcSet="img/desktop/1@2x.jpg 2x"
          />
          <p
            className="friends__name"
          >
            Katrine
          </p>
          <button
            className="friends__link"
          >
            Learn More
          </button>
        </li>
      </ul>
      <ul
        className="friends__pagination"
      >
        <li
          className="friends__page-item"
        >
          <button
            className="friends__button friends__button--active"
          >
            1
          </button>
        </li>
        <li
          className="friends__page-item"
        >
          <button
            className="friends__button"
          >
            2
          </button>
        </li>
        <li
          className="friends__page-item"
        >
          <button
            className="friends__button"
          >
            3
          </button>
        </li>
        <li
          className="friends__page-item"
        >
          <span
            className="friends__ellipsis"
          >
            ...
          </span>
        </li>
        <li
          className="friends__page-item"
        >
          <button
            className="friends__button"
          >
            10
          </button>
        </li>
      </ul>
    </section>
  </main>,
]
`;
